packages/smooth_app/lib/main.dart

```diff
@@ -173,6 +178,9 @@ class SmoothAppGetLanguage extends StatelessWidget {
       DefaultAssetBundle.of(context),
       languageCode,
     );
+    if (isFirstTimeUser) {
```

Comment:
Actually we would be better off storing the welcome page number.
This way we could test "which welcome page should I start with?".
In case there are 5 welcome pages and the user had to stop after welcome page 2. We shouldn't start again from welcome page 1, and we shouldn't dismiss either welcome pages 3, 4 and 5.

Reply:
hmm that's a good feature, in that case we can create an `OnboardingFlow` that first figures out which page the user stopped at last. I'll store it in userPref, can do it in the next PR, don't want to make this one extra large.